docs/post-deployment.rst

```diff
@@ -0,0 +1,2 @@
+Post Deployment
```

Comment:
@Ishiihara It seems that we can probably get rid of the post deployment section or do you have anything useful to say here? Take a look at Kafka/schema registry to see examples